src/awscli-asset.ts

```diff
@@ -1,18 +1,24 @@
 import * as path from 'path';
+import { ILambdaLayerAsset } from '@aws-cdk/interfaces';
 import { FileSystem } from 'aws-cdk-lib';
 import * as s3_assets from 'aws-cdk-lib/aws-s3-assets';
 import { Construct } from 'constructs';
 
 /**
  * A CDK Asset construct that contains the AWS CLI.
  */
-export class AwsCliAsset extends s3_assets.Asset {
+export class AwsCliAsset extends s3_assets.Asset implements ILambdaLayerAsset {
```

Comment:
Maybe call this `AwsCliV1Asset` or something.

Reply:
The package it's released in is called `@aws-cdk/asset-awscli-v1`. So I don't think that's necessary. Let me know if you still disagree.

Reply:
I know, but I like the explicitness in the final code.